### PR TITLE
fix: omit old PRs from X.0.0 release notes

### PR DESCRIPTION
**Pull request by ckerr**
#### Description of Change

In X.0.0 releases, skip PRs that have already been backported and released to older branches.

CC @codebytere because she's shown interest, but anyone is welcome to review

#### Checklist

- [x] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: script/release/notes/notes.js [hunk 1]
```diff
@@ -564,15 +511,17 @@ const getNotes = async (fromRef, toRef, newVersion) => {
   return notes;
 };
 
+// FIXME: Chromium commit messages don't have this info anymore...
```
Comment (jkleinsc) [inline script/release/notes/notes.js:514]: What changed here?  Is it something we are differently in our roller PRs?

Comment (ckerr) [reply script/release/notes/notes.js:514]: User error. I was testing this from `master` but should've been in `9-x-y` :roll_eyes:

Comment (ckerr) [reply script/release/notes/notes.js]: Incorrect comment removed in f04d6a5

**Comment by ckerr** [comment]
(rebased to master pick up https://github.com/electron/electron/pull/23266 to try to fix the lint error)

**Comment by ckerr** [comment]
argh

**Comment by ckerr** [comment]
Superceded by https://github.com/electron/electron/pull/23335

**Pull request closed without merging**